game.js

```diff
@@ -0,0 +1,86 @@
+var gamePattern = []; // global var array to store game pattern
+var userClickedPattern = []; // global var array to store user clicked pattern
+var level = 0; 
+var gameStarted = false;
+
+var game = () => {
+  $(document).on("keypress", () => {
+    if (!gameStarted) {
+      startGame();
+    }
+  });
+};
+
+var startGame = () => {
+  gameStarted = true;
+  playGameCard();
+};
+
+var playSound = (name) => {
+  $("body").append(
+    `<audio id=sound-${name}> <source src = "sounds/${name}.mp3" type = "audio/mpeg" > </audio>`
+  );
+  $(`audio#sound-${name}`)[0].play();
```

Comment:
With this implementation, we add a new <audio> element to the document every time we play a sound, and those elements stay in the document until the page is unloaded.

Could we instead have a single <audio> element for each sound in the HTML file that we can play repeatedly?

Reply:
A little more explanation on why this is the case here in case it's not obvious:

Whenever you're calling playSound(), you're basically adding an audio element to the DOM. Since you call that function everything a user plays, it adds a new element every time the user plays a card. The old elements are never removed.

What Kevin is suggesting here is that you add all possible sounds in different elements at the initialization of the game. Then, this function here wouldn't have to append an audio element as this element already exists. You would only have to call the play() function on that element of type audio with id sound-${name}, $(`audio#sound-${name}`).